.github/workflows/e2e.yml

```diff
@@ -1,33 +1,43 @@
 name: E2E tests
 on:
+  workflow_call:
+  workflow_dispatch:
```

Comment:
**Nitpick:** I think we only need `workflow_call` to be able to call this workflow from the deployment one. `workflow_dispatch` is only necessary if we want to trigger it manually.

**Thought:** We can also run it on a schedule (e.g. daily) by using the [schedule directive](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule). Perhaps in that case we should send the results to Slack somehow? Or report them somewhere else that we're aware?

Reply:
Schedule and alert on slack is good idea.
I've added workflow_dispatch just in case. But if you want I can remove it

Reply:
No, it's fine that it's there. Perhaps we want to run the tests manually at some point. :slightly_smiling_face: